server/controllers/apply/applicationsController.test.ts

```diff
@@ -38,7 +38,8 @@ describe('applicationsController', () => {
 
   describe('index', () => {
     it('renders the index view', async () => {
-      const applications = applicationFactory.buildList(5)
+      const applications = { inProgress: [], requestedFurtherInformation: [], submitted: [] } as GroupedApplications
```

Comment:
I could be wrong but I don't think we need to cast here, it would be better if we could do `const applications: GroupedApplication = ...`

Reply:
Yeah, much better!